ospfd/ospfd.c

```diff
@@ -972,7 +980,7 @@ struct ospf_area *ospf_area_new(struct ospf *ospf, struct in_addr area_id)
 	ospf_opaque_type10_lsa_init(new);
 
 	new->oiflist = list_new();
-	new->ranges = route_table_init();
+	new->ranges = route_table_init_with_delegate(&ospf_range_table_delegate);
 	new->nssa_ranges = route_table_init();
```

Comment:
Should new->nssa_ranges = route_table_init() do the same new init as new->ranges?